src/ln.js

```diff
@@ -35,6 +35,10 @@ function _ln(options, source, dest) {
   var isAbsolute = (path.resolve(source) === sourcePath);
   dest = path.resolve(process.cwd(), String(dest));
 
+  if (fs.existsSync(dest) && common.statFollowLinks(dest).isDirectory(dest)) {
+    dest = path.resolve(dest, path.basename(sourcePath));
```

Comment:
I think this should be `path.join()`. The docs for `path.resolve()` claim it produces an absolute path. `ln -s` actually cares if it's passed an absolute path vs. relative path, however, and `path.join()` preserves that.

Reply:
Done. I've replaced `resolve` with `join`. Shouldn't change anything, but it doesn't hurt either. 😁

Reply:
Ah, you're right. It's ok to `resolve` the destination, but not OK to resolve the source. Either `join` or `resolve` is fine.